ts/index.ts

```diff
@@ -1,25 +1,4 @@
-import * as c3 from 'c3';
-import {unpack} from 'msgpackr/unpack';
+import { displayChart, displayHeader } from './page-load';
```

Comment:
I added `object-curly-spacing` to eslintrc to enforce the old curly brace style
please rebase and run `yarn lint`